website/docs/release_notes.md

```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
+
+```
+ //> using test.dep "org.scalameta::munit::0.7.29"
+ //> using test.jar "path/to/jar"
+ //> using test.javaOpt "-Dfoo=bar"
+ //> using test.javacOpt "source", "1.8", "target", "1.8"
+ //> using test.javaProp "foo1=bar1"
+ //> using test.option "-Xfatal-warnings"
+ //> using test.resourceDir "testResources"
+ //> using test.toolkit "latest"
+```
+
+Added by [@Gedochao](https://github.com/Gedochao) in  [#2046](https://github.com/VirtusLab/scala-cli/pull/2046)
+
+### Changes to using-directives syntax
+
+We've made several updates to simplify the using directives syntax in this release:
+
+- removed the possibility of nesting from the API.
+- allowed omitting commas in lists of values.
+- disallowed multiline comments.
+- removed multiline strings.
+- removed `require` and `@require` syntax support.
+- allowed values without quotes.
+- removed `@using`.
+
+
+Added by [@tgodzik](https://github.com/tgodzik) in [#2076](https://github.com/VirtusLab/scala-cli/pull/2076)
+
+
+### Bootstrapped standalone fat JAR.
+
+The Scala CLI launcher is available as a standalone fat JAR. You can download the nightly version of the Scala CLI fat JAR from Sonatype and try it now:
+
+```bash ignore
+cs launch org.virtuslab.scala-cli:cliBootstrapped:1.0.0-RC2 -M scala.cli.ScalaCli
+```
+
+Added by [@romanowski](https://github.com/romanowski) in [#2005](https://github.com/VirtusLab/scala-cli/pull/2005).
+
+
+### Access the Name of the Running Script
+
+With the special `scriptPath` function, you can now easily access the name of the running script inside the script itself. Example of how to use the scriptPath variable:
+
+```scala title=scripts/hello.sc
+#!/usr/bin/env -S scala-cli shebang
+
+println(scriptPath)
+```
+
+<ChainedSnippets>
+
+```bash
+chmod +x scripts/hello.sc
+./scripts/hello.sc
+```
+
+```text
+./scripts/hello.sc
+```
+
+</ChainedSnippets>
+
+Added by [@lwronski](https://github.com/lwronski) in [#1990](https://github.com/VirtusLab/scala-cli/pull/1990)
+
+### Explicit Handling of Paths in using-directives
+
+The `${.}` pattern in directive values can now be replaced by the parent directory of the file containing the directive. This makes it possible to generate coverage output files relative to the source file location, for example:
+
+```
+//> using options "-coverage-out:${.}"
+```
+
+Added by [@lwronski](https://github.com/lwronski) in [#2040](https://github.com/VirtusLab/scala-cli/pull/2040)
+
+### Fix deadlocks in Script Wrappers
+
+We have resolved an issue that caused deadlocks when threads were run from the static initializer of the wrapper object 
+([#532](https://github.com/VirtusLab/scala-cli/pull/532) and [#1933](https://github.com/VirtusLab/scala-cli/pull/1933)).
+Based on the feedback from the community (Thanks [@dacr](https://github.com/dacr)), we found that encapsulating the script code
+into a class wrapper fixes the issue. The wrapper is generated by the Scala CLI and is not visible to the user.
```

Comment:
It might be worth mentioning the behaviour change when `@main` is used, maybe with an output error example